web-client/integration-tests/caseServicesSupervisorJourney.test.ts

```diff
@@ -23,6 +29,38 @@ describe('Case Services Supervisor Messages Journey', () => {
     cerebralTest.closeSocket();
   });
 
+  loginAs(cerebralTest, 'petitioner@example.com');
+  it('Creates the first case', async () => {
+    const caseDetail = await uploadPetition(cerebralTest, {
+      contactPrimary: {
+        address1: '734 Cowley Parkway',
+        city: 'Amazing',
+        countryType: COUNTRY_TYPES.DOMESTIC,
+        name: 'welcome to flavortown',
+        phone: '+1 (884) 358-9729',
+        postalCode: '77546',
+        state: 'AZ',
+      },
+    });
+
+    expect(caseDetail.docketNumber).toBeDefined();
+    cerebralTest.docketNumber = caseDetail.docketNumber;
+  });
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
+  docketClerkCreatesAnOrder(cerebralTest, {
+    documentContents: embedWithLegalIpsumText('magic'),
+    documentTitle: 'some title',
+    eventCode: 'O',
+    expectedDocumentType: 'Order',
+    signedAtFormatted: '01/02/2020',
+  });
+  docketClerkSignsOrder(cerebralTest);
+  docketClerkAddsDocketEntryFromOrder(cerebralTest, 0);
+
+  // TODO: add a docket entry work item associated with docket section
+  // TODO: add a docket entry work item associated with petitions clerk section
```

Comment:
Just checking that these are left intentionally

Reply:
They are done, thanks for noticing!